lib/relations_sql/expression.py

```diff
@@ -160,18 +160,18 @@ def generate(self, **kwargs):
         self.args = []
 
 
-class SCHEMA(NAME):
+class SCHEMANAME(NAME):
```

Comment:
Why is the naming changing?

Reply:
Great question! Because it's not actually a schema, but a schema name. See response below. In this case, with the change with TABLENAME and COLUMNNAME, this is my consistent.